### DISCUSSION TITLE
Can't get PC to turn on.

### DISCUSSION THREAD

**Discussion by TetsujinXLIV** [AlexGustafsson/homebridge-wol#212] [Questions]
I verified WOL works on this computer. I'm able to turn this PC on using WOL from my main computer. But when I try to do it with this it's not working. One thing to note I'm not sure if it matters but when I use this WOL program on windows I have to select local subnet instead of the internet option to make it work. This is my config I am using on on the plugin:

"name": "LinuxComputer",
"manufacturer": "homebridge-wol",
"model": "NetworkDevice",
"host": "111.111.111.111",
"mac": "xx:xx:xx:xx:xx:xx",
"broadcastAddress": "255.255.255.255",
"pingInterval": 45,
"pingsToChange": 5,
"pingTimeout": 1,
"pingCommandTimeout": 0,
"startCommandTimeout": 2,
"wakeGraceTime": 10,
"wakeCommandTimeout": 0,
"logLevel": "Debug",
"returnEarly": false,
"accessory": "NetworkDevice"

One thing I am wondering is I have 2 network devices on my setup both net1 and eth0. In homebridge eth0 doesn't work because it uses the wrong IP so I setup homebridge to use net1 and that fixes homebridge. Could this be causing part of the problem?

**Comment by AlexGustafsson** [comment]
The plugin shouldn't be sensitive to network interfaces. For example, if you've specified 192.168.1.111 as the host, the interface with an address on the 192.168.1.0/24 network will be used. But for WoL packages, things are a bit different I believe. It's possible that it is being broadcasted on the wrong interface. As you've specified the broadcast address to be 255.255.255.255, I suppose all interfaces should be used, though. The package we use for WoL is https://github.com/agnat/node_wake_on_lan, it includes a utility to test it. You could try it out to see if it works for you from your server running Homebridge. That way we can hopefully rule something out at least.
